api/src/main/java/org/openmrs/module/locationbasedaccess/utils/LocationUtils.java

```diff
@@ -61,4 +61,16 @@ public static Boolean doesUsersForPersonBelongToGivenLocation(Person person, Str
     public static Boolean compare(String value1, String value2) {
         return (StringUtils.isNotBlank(value1) && StringUtils.isNotBlank(value2)) && value1.equals(value2);
     }
+
+    public static String getUserAccessibleLocationUuid(User authenticatedUser) {
+        String accessibleLocationUuid = null;
+        if (authenticatedUser != null) {
```

Comment:
To reduce nesting, you could just return if authenticatedUser is null

Reply:
Updated